### PR TITLE
[W6.1h][W14-B2]Teo Wen Zong

### PR DESCRIPTION
**Pull request by wenzongteo**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Phone.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * Represents a Person's phone number in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
  */
-public class Phone {
+public class Phone implements Printable{
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Phone.java:9]: need space between `Printable {`

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:3]: Need header comments for the interface and the method.

File: src/seedu/addressbook/data/person/ReadOnlyPerson.java [hunk 3]
```diff
@@ -66,20 +66,31 @@ default String getAsTextShowAll() {
...
         for (Tag tag : getTags()) {
             builder.append(tag);
         }
         return builder.toString();
     }
+
+    /**
+     *  Returns the final String of the details of each person.
+     */
+    default String getPrintableString(Printable... printable){
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/ReadOnlyPerson.java:89]: ideally you should use this method somewhere

**Comment by okkhoy** [comment]
@wenzongteo 
Some comments added. Please ack & close the PR after reading comments.

**Comment by wenzongteo** [comment]
@okkhoy alright! Will take note to follow the coding convention strictly and also ensure necessary comments are created

**Pull request closed without merging**